biblio.rst

```diff
@@ -21,18 +21,12 @@ Bibliography
 .. [Mez07]
     F. Mezzadri, *How to generate random matrices from the classical compact groups*, Notices of the AMS **54** 592-604 (2007). :arxiv:`math-ph/0609050`.
 
-.. [Mis12]
```

Comment:
Did these references get removed because they're not used? If so, perhaps we could use them somewhere instead? It seems a shame to lose useful links to background information (e.g. I might want to read these sometime).

Reply:
They're not referenced in the code.  You can check the `git log` (e.g. `git log -S Mis12`) to see how they came and went, but my recollection from when I did that is that a couple of them were added and never used, and one was used in documentation that got deleted in something like ~2017.

Reply:
I think I would still vote to reference them rather than delete them entirely unless they are clearly irrelevant. The reference can even just be a "here are some other references" or "further reading" type statement at the end. Otherwise perhaps we can list them in the existing text alongside the most similar seeming paper.

Reply:
I mean, what makes these references any more valid for inclusion than any of the million other published papers that also aren't referenced here?  The main qualification is just "these happened to be in Rob's Mendeley back in 2011 and got accidentally committed".

Absolutely happy to add more references if they're referenced in the text, but I removed these ones here because they're not referred to in the text, and this PR was (in theory!) just about fixing all the warnings.  It's not that they're accidentally not referenced, it's just they weren't removed from here when their text left the guide.

Reply:
Because Rob Mendeley bothered to type them here? :)

But I'm not entirely against them being removed, so I'm okay with them going.

Reply:
Fair enough!  I'll leave them out now, because adding them in would mean that I'd have to read the papers and find a suitable place to cite them in the text.